core/drivers/crypto/hisilicon/sec_hash.c

```diff
@@ -335,14 +335,14 @@ TEE_Result hisi_sec_digest_do_update(struct hashctx *hash_ctx,
 		hash_ctx->in = malloc(hash_ctx->buf_len);
 		if (!hash_ctx->in) {
 			EMSG("Fail to alloc in data buf");
-			return TEE_ERROR_STORAGE_NO_SPACE;
+			return TEE_ERROR_OUT_OF_MEMORY;
```

Comment:
Could you move change in core/drivers/crypto/hisilicon/sec_hash.c in a dedicated commit for this fixes. That could should include a `Fixes:` tag:
`Fixes: 94c8a3397ec4 ("core/drivers/crypto/hisilicon/sec_hash.c")`

Reply:
OK

Reply:
copy/paste error in my comment above: the Fixes tag should be:
`Fixes: 94c8a3397ec4 ("drivers: crypto: hisilicon:Add HASH and HMAC algorithm")`